packages/db/prisma/prismaClient.ts

```diff
@@ -13,15 +13,6 @@ export const prismaClient: PrismaClient =
 
 if (process.env.NODE_ENV !== "production") global.prismaClient = prismaClient;
 
-export const prismaLogClient = prismaClient.$extends({
-  query: {
-    beforeAll: async (params) => {
```

Comment:
breaks in prisma's new version, still needed? @ogroppo

Reply:
can be removed and added later yes